src/main/java/ita/softserve/course_evaluation/controller/GroupController.java

```diff
@@ -0,0 +1,96 @@
+package ita.softserve.course_evaluation.controller;
+
+import ita.softserve.course_evaluation.dto.GroupDto;
+import ita.softserve.course_evaluation.dto.GroupDtoMapper;
+import ita.softserve.course_evaluation.entity.Group;
+import ita.softserve.course_evaluation.service.GroupService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+
+import java.util.List;
+
+@RestController
+public class GroupController {
+
+    @Autowired
+    private GroupService groupService;
+
+    @Autowired
```

Comment:
```@Autowired``` seems like obsolete, because Spring Framework since version 4.3 do this behind the scenes via the constructor, that you have declared on line 24-27. Also, we do not inject into field directly